capstone-frontend/src/BookSearchTile.test.js

```diff
@@ -39,7 +39,7 @@ test('renders book search tile test', () => {
   const bookAuthor = document.getElementsByClassName("book-authors")[0];
   expect(bookAuthor).toBeInTheDocument();
 
-  const bookListButton = document.getElementById("dropdown-list-add");
+  const bookListButton = document.getElementById("button-list-add");
```

Comment:
nit: Use single quotes for string literals (https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes)

Reply:
Fixed string literals in my edited files: this test file, BookPage and BookListAddDropdown